new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java

```diff
@@ -32,7 +78,18 @@ public static List listDeployments() {
     // --------------------------- namespaces -----------------------------------
 
     public static boolean createNamespace(String name) {
-        return true;
+        V1ObjectMeta meta = new V1ObjectMeta();
+        meta.name(name);
+        V1Namespace namespace = new V1Namespace();
+        namespace.metadata(meta);
+        try {
+            namespace = coreV1Api.createNamespace(namespace, pretty, null, null);
+            System.out.println("Kubernetes.createNamespace namespace: " + namespace);
+            return true;
+        } catch (ApiException e) {
+            e.printStackTrace();
```

Comment:
probably a good idea to return the exception

Reply:
@markxnelson Should we catch Throwable throughout ?

Reply:
I prefer to catch the most specific object and handle it

Reply:
I realize that's SOP for system programming in Java to narrowly only check for the checked exceptions.  But I have two counter thoughts to this.  One, we're talking about testing here - so 'happy path' programming is far more acceptable.   Two, if there's an insistence not to catch Runtime/Throwable throughout, then there will be many cases where an exception will float out of a library all the up to the highest level.  But I thought the goal you set out was to prevent this from happening.  For example, there's a (sad IMO) school of thought these days that even `normal` paths can normally throw runtimes - even some JDBC drivers do that unfortunately - and we've no clue in a callout to external library what the hell it's going to throw...

Reply:
Even in system programming catch Throwable tends to be useful.  For calls to the 'outside' we've found we had to do that in WL in many places.  Calling customer code.  Calling shaky external libraries.

Reply:
Hi Mark, can you clarify here?  I assume you want to just declare the exception in the throws clause and have it propagate up?   Just a little confusion on my part relative to the discussion about handling the types of exceptions.  Thanks for clarifying.

Reply:
yeah, i am thinking about the test in the calling method where we say something like ```
```assert.DoesNotThrow(whatever(), ApiException.class, e,
  String.format("could not do whatever, got exception %s", e))```

Reply:
Added throws clause to method signature.